### ISSUE TITLE
Deleting a Node does not remove the associated MQTT topic

### ISSUE THREAD

**Issue by blhoward2** [OpenZWave/qt-openzwave#169]
After deleting a Node in ozwadmin, MQTT Explorer shows that the topic still existed within MQTT. The device didn't disappear automatically from HA, which I suspect is linked to this behavior.

**Comment by ajschmidt8** [comment]
Dropping a StackOverflow link here for reference: https://stackoverflow.com/questions/42218056/how-to-remove-a-published-topic

I'm not sure if removing a topic is a thing.

**Comment by blhoward2** [comment]
If true, then there must be a way to invalidate the payload so that HA knows the entity is deleted. Or maybe it’s just removing the retained flag.

**Comment by psgcooldog** [comment]
Use MQTT Explorer to manually delete the dead node.

**Comment by blhoward2** [comment]
Yes, I fixed my setup months ago when I posted this. That doesn’t change the fact that it’s a bug and that this should happen automatically.
